MokManager.c

```diff
@@ -1961,25 +1970,28 @@ static EFI_STATUS mok_key_enroll(void)
 	EFI_FILE *file = NULL;
 	UINTN filesize;
 	void *data;
-
-	simple_file_selector(&im, (CHAR16 *[]){
-      L"Select Key",
-      L"",
-      L"The selected key will be enrolled into the MOK database",
-      L"This means any binaries signed with it will be run without prompting",
-      L"Remember to make sure it is a genuine key before Enroling it",
-      NULL
-	      }, L"\\", L"", &file_name);
+	CHAR16 *selections[] = {
+		L"Select Key",
+		L"",
+		L"The selected key will be enrolled into the MOK database",
+		L"This means any binaries signed with it will be run without prompting",
+		L"Remember to make sure it is a genuine key before Enroling it",
```

Comment:
Same as above: 'Enroling'.

Reply:
These were already that way, of course, but I've fixed them up in the next revision anyway.